### ISSUE TITLE
cds translate error

### ISSUE THREAD

**Issue by hyphaltip** [stajichlab/Phyling#13]
this an error message - it would be helpful if this also included a report of what the sequences name is causing the problem (and sequence file) - I assume there is an internal stop codon, it would be better to just skip failed sequences than crash the whole thing.

```
2023-10-05 10:17:52,338 PHYling INFO Inputs are dna sequences
Traceback (most recent call last):
  File "/bigdata/stajichlab/shared/condaenv/phyling2/bin/phyling", line 8, in <module>
    sys.exit(main())
  File "/bigdata/stajichlab/shared/condaenv/phyling2/lib/python3.10/site-packages/phyling/phyling.py", line 208, in main
    args.func(**vars(args))
  File "/bigdata/stajichlab/shared/condaenv/phyling2/lib/python3.10/site-packages/phyling/libphyling.py", line 431, in main
    msa = msa_generator(inputs)
  File "/bigdata/stajichlab/shared/condaenv/phyling2/lib/python3.10/site-packages/phyling/libphyling.py", line 155, in __init__
    self._pep_seqs = self._sequences.translate()
  File "pyhmmer/easel.pyx", line 5835, in pyhmmer.easel.DigitalSequenceBlock.translate
  File "pyhmmer/easel.pyx", line 5889, in pyhmmer.easel.DigitalSequenceBlock.translate
ValueError: Incomplete sequence of length 1672 at index 2281
```

**Comment by chtsai0105** [comment]
This error is due to the cds sequence cannot be divided by 3, as described in [pyhmmer](https://pyhmmer.readthedocs.io/en/stable/api/easel.html#pyhmmer.easel.DigitalSequenceBlock.translate)
```
1672 / 3 = 557.333333...
```
Xinzhan already reported this issue and I'm trying to fix this by catching the ValueError raised by pyhmmer, skip the sequence and cast an warning of the problematic filename/seuqence.

**Comment by chtsai0105** [comment]
Fix by fe7267fb2ba26062cf70f8dd6d35c320b325a585
